src/engine/client/client.cpp

```diff
@@ -509,6 +511,12 @@ void CClient::EnterGame()
 	// to finish the connection
 	SendEnterGame();
 	OnEnterGame();
+	
+	// tell the server that this is a netgui compatible client
+	CNetMsg_Cl_NetGui_TriggerEvent NGMsg;
+	NGMsg.m_Type = 1883;
```

Comment:
You can remove this. If the client is compatible with ModAPI, he is compatible with NetGUI